packages/ra-core/src/auth/useLogoutIfAccessDenied.spec.tsx

```diff
@@ -79,56 +76,67 @@ describe('useLogoutIfAccessDenied', () => {
     });
 
     it('should not logout if passed no error', async () => {
-        const { queryByText } = renderInRouter(
-            <AuthContext.Provider value={authProvider}>
-                <TestComponent />
-            </AuthContext.Provider>
-        );
+        render(<Route path="/" element={<TestComponent />} />, {
```

Comment:
Why do we need a route?

Reply:
Because `useLogoutIfAccessDenied` calls `useNavigate` and that requires to be inside a Router